Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.yml

```
@@ -13,14 +13,52 @@ args:
 - name: html
   required: true
   default: true
-  description: The HTML to strip tags from
+  description: The HTML to strip tags from.
 - name: html_tag
   default: true
   defaultValue: body
   description: Specify HTML tag to extract the text from within.
+- name: allow_body_fallback
+  default: true
+  auto: PREDEFINED
+  predefined:
+  - "true"
+  - "false"
+  description: 'Allow using the input HTML as a fallback for the Body, if no body-tag is found (default: false).
```

```suggestion
  description: 'Allow using the input HTML as a fallback for the Body, if no body-tag is found.
```

We don't write default values in the description by convention. This is already added automatically in the UI (which determines the default type from the `default` key).